app/code/Ludmila/LSAskQuestion/view/adminhtml/ui_component/all_questions.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<listing xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+    <argument name="data" xsi:type="array">
+        <item name="js_config" xsi:type="array">
+            <item name="provider" xsi:type="string">all_questions.all_questions_data_source</item>
+        </item>
+    </argument>
+    <settings>
+        <spinner>askquestion_columns</spinner>
+        <deps>
+            <dep>all_questions.all_questions_data_source</dep>
+        </deps>
+    </settings>
+    <dataSource name="all_questions_data_source" component="Magento_Ui/js/grid/provider">
+        <settings>
+            <storageConfig>
+                <param name="indexField" xsi:type="string">question_id</param>
+            </storageConfig>
+            <updateUrl path="mui/index/render"/>
+        </settings>
+        <aclResource>Magento_Backend::admin</aclResource>
+        <dataProvider class="Magento\Framework\View\Element\UiComponent\DataProvider\DataProvider" name="all_questions_data_source">
+            <settings>
+                <requestFieldName>id</requestFieldName>
+                <primaryFieldName>question_id</primaryFieldName>
+            </settings>
+        </dataProvider>
+    </dataSource>
+    <listingToolbar name="listing_top">
+        <settings>
+            <sticky>true</sticky>
+        </settings>
+        <bookmark name="bookmarks"/>
+        <columnsControls name="columns_controls"/>
+        <filterSearch name="fulltext"/>
+        <filters name="listing_filters"/>
+        <exportButton name="export_button"/>
+
+        <massaction name="listing_massaction">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="indexField" xsi:type="string">question_id</item>
+                </item>
+            </argument>
+
+            <action name="change_status">
+
+                <settings>
+                    <url path="allquestions/question/massStatus"/>
+                    <type>status</type>
+                    <label translate="true">Change status to Answered</label>
```

Comment:
ok, I understand why massaction was changed. But it is possible that admin wanted to change the status to different one?